torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3253,8 +3344,8 @@ def reference_lgamma(x):
                    handles_complex_extremals=False),
     UnaryUfuncInfo('square',
                    ref=np.square,
-                   dtypes=all_types_and_complex_and(torch.bool),
-                   dtypesIfCPU=all_types_and_complex_and(torch.bool),
+                   dtypes=all_types_and_complex_and(torch.bool, torch.half, torch.bfloat16),
+                   dtypesIfCPU=all_types_and_complex_and(torch.bool, torch.half, torch.bfloat16),
                    dtypesIfCUDA=all_types_and_complex_and(torch.bool, torch.half, torch.bfloat16),
```

Comment:
@heitorschueroff, `dtypes` is now same as `dtypesIfCPU` and `dtypesIfCUDA`. But if I remove any of the latter 2, then some `test_unsupported_dtypes` start failing.